source/simulator/core.cc

```diff
@@ -2036,6 +2048,10 @@ namespace aspect
             goto start_time_iteration;
           }
 
+        // invalidate the value of pre_refinement_step since it will no longer be used from here on
+        if ( timestep_number == 0 )
+        pre_refinement_step = std::numeric_limits<unsigned int>::max();
```

Comment:
This looks like the wrong indentation. Did you run the indentation script on this, or is github simply not showing the indentation correctly?